WooCommerce/WooCommerceTests/ViewRelated/Orders/OrderListViewModelTests.swift

```diff
@@ -241,7 +241,7 @@ final class OrderListViewModelTests: XCTestCase {
 
     // MARK: - Banner visibility
 
-    func test_when_having_no_error_and_orders_banner_should_not_be_shown_then_shows_nothing() {
+    func test_when_having_no_error_and_banner_should_not_be_shown_then_shows_nothing() {
```

Comment:
```suggestion
    func test_banner_should_not_be_shown_when_there_is_no_error() {
```

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/11582/commits/f22843e06fdfa86230abd876d16c8a52ecb0e97c